README.md

```diff
@@ -51,7 +55,7 @@ Our current student cohort includes:
   - Majors: Ecology & Evolutionary Biology and Optical Sciences & Engineering
 
 ## Research Collaborators
-In addition to our core research applications development team, we've had a number of collaborators across AIR programs and affiliated groups across the University of Arizona. 
+In addition to our core research team, we've had a number of collaborators across AIR programs and affiliated groups across the University of Arizona. 
```

Comment:
Maybe with some upcoming projects we also mention outside collaborators?

Reply:
I think that's a good idea. We can list ALERTWest and B2. Are there others you can think of that we should highlight @tweissaz ?

Reply:
Nah, all I can think of for now.